mantle/platform/api/aws/ec2.go

```diff
@@ -120,6 +120,16 @@ func (a *API) CreateInstances(name, keyname, userdata string, count uint64) ([]*
 	if keyname == "" {
 		key = nil
 	}
+
+	var rootBlockDev []*ec2.BlockDeviceMapping
+	if minDiskSize > 0 {
+		rootBlockDev = append(rootBlockDev, &ec2.BlockDeviceMapping{
+			DeviceName: aws.String("/dev/xvda"),
+			Ebs: &ec2.EbsBlockDevice{
+				VolumeSize: &minDiskSize,
+			},
+		})
+	}
```

Comment:
How does this affect tests that don't specify `minDiskSize` ? i.e. is specifying an empty `BlockDeviceMappings:` the same thing as what we were doing before?

Reply:
Yup, exactly! Since by golang rules the default value for `MinDiskSize` is 0 and the default value for the `BlockDeviceMappings` is an empty slice, nothing should change for other tests.